docs/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Document</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="description" content="Description">
```

Comment:
Maybe adding something more meaningful for SEO, such as:

`<meta name="description" content="RapiDAST: A tool for dynamic application security testing (DAST)">`

Reply:
updated